### PR TITLE
Translation into Brazilian Portuguese and corrections to docs in English

### PR DESCRIPTION
**Pull request by luizcmarin**
Note the following:
- created folders according to the standard for documentation
- all links checked

### CODE REVIEW

File: psalm80.min.xml
Comment (vjik) [inline psalm80.min.xml:1]: This file not need.

**Review by vjik — CHANGES_REQUESTED** [review]

File: psalm80.min.xml
Comment (luizcmarin) [reply psalm80.min.xml:1]: When I did this I had the minify plugin activated 🤦

**Comment by samdark** [comment]
See https://github.com/yiisoft/yii-console/actions/runs/8781505107/job/24093666268?pr=202

**Review by samdark — CHANGES_REQUESTED** [review]
Please check tests.

**Pull request merged**